app/templates/snippets-management/index.hbs

```diff
@@ -0,0 +1,109 @@
+{{page-title (t "snippets.manage-snippet-lists")}}
+<AuToolbar @size="medium" @skin="tint" @border="bottom" as |Group|>
+  <Group>
+    <ul class="au-c-list-horizontal au-c-list-horizontal--small">
+      <li class="au-c-list-horizontal__item">
+        <AuLink @route="index">
+          {{t "utility.back-to-overview"}}
+        </AuLink>
+      </li>
+      <li class="au-c-list-horizontal__item">
+        {{t "snippets.manage-snippet-lists"}}
+      </li>
+    </ul>
+  </Group>
+</AuToolbar>
+<div class="au-c-body-container au-c-body-container--scroll">
+  <AuDataTable
+    @content={{this.model}}
+    @fields="Label"
+    @noDataMessage={{t "snippets.crud.no-data"}}
+    @size={{this.size}}
+    @page={{this.page}}
+    @sort={{this.sort}} as |s|
+  >
+    <s.menu as |menu|>
+      <menu.general>
+        <AuToolbar class="au-o-box">
+          <AuToolbar::Group>
+            <AuHeading @skin="2">
+              {{t "snippets.manage-snippet-lists"}}
+            </AuHeading>
+          </AuToolbar::Group>
+          <AuToolbar::Group>
+            <AuButton
+              @skin="primary"
+              {{on "click" (perform this.createSnippetList)}}
+              @loading={{this.createSnippetList.isRunning}}
+            >
+              {{t "snippets.make-new"}}
+            </AuButton>
+          </AuToolbar::Group>
+        </AuToolbar>
+      </menu.general>
+    </s.menu>
+    <s.content as |c|>
+      <c.header>
+        <AuDataTableThSortable
+          @field="label"
+          @currentSorting={{this.sort}}
+          @label={{t "snippets.attr.label"}}
+        />
+        <th>{{t "snippets.attr.id"}}</th>
+        <th></th>
+      </c.header>
+      <c.body as |snippet|>
+        <td>
+          <AuLink
+            @skin="primary"
+            @route="snippets-management.edit"
+            @model={{snippet.id}}
+          >
+            {{#if snippet.label}}
+              {{snippet.label}}
+            {{else}}
+              Unnamed
+            {{/if}}
```

Comment:
Can be removed now I think?

Reply:
There's still a way of getting unnamed snippet lists if you create a snippet and you leave the page without saving your title

Reply:
You could fix that by
- when creating new snippet list, not saving it directly
- when updating the title, saving the whole snippet list (you already do this I think). That way if it is a new snippet, it will also create it, if it was just a title update, it will just update the title.
- to avoid garbage store data, remove this unsaved snippet list (will have record.isNew = true) in e.g. destroy function of the component, or wherever you have access to this new snippet (maybe even the route?). Either way you can easily clean up with rollbackAttributes.